custom/icds/location_reassignment/download.py

```diff
@@ -254,3 +261,65 @@ def _build_rows(self, site_code):
                 household_case.case_id
             ])
         return rows
+
+
+class OtherCases(object):
+    """
+    Download other cases that are not households or child cases of households
+    """
+    valid_operations = [SPLIT_OPERATION, EXTRACT_OPERATION]
+    headers = [NEW_NAME, NEW_SITE_CODE_COLUMN, CASE_NAME, CASE_ID_COLUMN]
+
+    def __init__(self, domain):
+        self.domain = domain
+
+    def dump(self, transitions):
```

Comment:
Checking my understanding: there won't be any household cases in this dump because the `transitions` passed in won't contain any household cases, is that right?

Reply:
The transition object does not have details of cases.
We ignore household cases when we fetch case ids later for dumping
https://github.com/dimagi/commcare-hq/pull/27480/files#diff-af1942757ada9f313046092629aac433R307